google-cloud-firestore/test/google/cloud/firestore/conformance_test.rb

```diff
@@ -208,21 +208,22 @@ class ConformanceDelete < ConformanceTest
   def self.build_test_for description, test, file_path
     define_method("test_#{file_path}: #{description}") do
       doc_ref = doc_ref_from_path test.doc_ref_path
-      opts = {}
+      exists = nil
+      update_time = nil
       if test.precondition && test.precondition.exists
-        opts[:exists] = test.precondition.exists
+        exists = test.precondition.exists
       end
       if test.precondition && test.precondition.update_time
-        opts[:update_time] = Time.at(test.precondition.update_time.seconds)
+        update_time = Time.at(test.precondition.update_time.seconds)
```

Comment:
Drop the parens? Seems like these files aren't covered by Rubocop...

Reply:
That's right, test files in veneers are **very unfortunately** not covered by rubocop.

Reply:
Fixed, thanks!